glslc/README.asciidoc

```diff
@@ -236,6 +236,66 @@ that all `warning:` messages are shown as `error:` and any warnings will cause
 a non-zero exit code from `glslc`. If `-w` is specified the warnings
 generated are suppressed before they are converted to errors.
 
+=== Dependency Generation Options
+
+==== `-M` or `-MM`
+
+`-M` generates *make* dependencies, outputs a rule suitable for *make*
+describing the dependencies of the main source file. Instead of outputting
+the result of preprocessing, the preprocessor outputs one *make* rule
+containing the SPIR-V object file name for that source file, a colon, and the
+names of all the included files.
+
+Unless specified explicitly (with `-MT`), the SPIR-V object file name in the
+generated *make* rules follows the rules of  <<output-file-naming,Output File
+Naming>> as in `-c` compilation stage.
+
+The dependency info file name can be specified by `-o` and `-MF` options. When
+both are specified, `-o` option is ignored.
+
+Passing `-M` implies `-E`, and suppresses warnings with an implicit `-w`.
+
+`-MM` is an alias of `-M`.
```

Comment:
I think it reads better as "alias for" instead of "alias of"  ??
Not too worried about this, though.